plugins/CoreHome/templates/_menu.twig

```diff
@@ -45,15 +45,13 @@
                 {% endfor %}
 
                 {% if hasSubmenuItem %}
-                    <li id="{% if level2._url is defined and level2._url is not empty %}{{ _self.getId(level2._url) }}{% endif %}" class="menuTab">
-
+                    <li role="menuitem" class="menuTab" title="{{ level1|translate|e('html_attr') }}" id="{% if level2._url is defined and level2._url is not empty %}{{ _self.getId(level2._url) }}{% endif %}">
                         <a class="item" href="">
-                            <span class="menu-icon {{ level2._icon|default('icon-arrow-right') }}"></span>{{ level1|translate }}
-                            <span class="hidden">
-                             {{ 'CoreHome_Menu'|translate }}
-                           </span>
+                            <span class="menu-icon {{ level2._icon|default('icon-arrow-right') }}"></span>
+                            {{ level1|translate }}
```

Comment:
Sorry to bother you again, I noticed some UI tests are failing now because the `{{ level1|... }` is no longer directly attached to the `</span>` as it was the case before.

This results in a whitespace between the icon and the menu name and so the menu item names are no longer aligned

![menus_admin_loaded](https://cloud.githubusercontent.com/assets/273120/10979783/569a482a-8463-11e5-830d-822d70c0c7e9.png)

BTW: Are you familiar with [squashing commits](http://eli.thegreenplace.net/2014/02/19/squashing-github-pull-requests-into-a-single-commit/)?